### DISCUSSION TITLE
Support for Swift 5.9.2

### DISCUSSION THREAD

**Discussion by eduardboamba** [github/codeql#15215] [Q&A]
I see currently only up to 5.9.1 is listed [here](https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/). Is support coming for 5.9.2 as well or is it perhaps already supported, but the documentation needs a slight update?

**Comment by jketema** [comment]
Hi @eduardboamba,

I'll check with the engineering team and report back.

**Reply by eduardboamba** [reply]
Just noticed the same for Kotlin. Currently supported up to 1.9.20, while 1.9.21 is out and 1.9.22 will soon follow.
Could you check it too or it requires a separate issue?
Cheers!

**Reply by jketema** [reply]
I'll check that too.

**Reply by jketema** [reply]
Kotlin 1.9.21 and 1.9.22 should work. The documentation indeed needs updating.

**Comment by jketema** [comment]
Regarding Swift: 5.9.2 likely works, but we have not yet extensively tested this. Hence, there's a small chance there are some unexpected issues, although not likely given that it is a point release. We'll update the documentation once we've been able to do some more extensive testing.
